.evergreen/config.yml

```diff
@@ -916,33 +920,22 @@ buildvariants:
     tasks:
       - name: "test-mlaunch"
 
-  - matrix_name: "mongo-4.7"
-    matrix_spec:
-      auth-and-ssl: ["auth-and-ssl", "noauth-and-nossl"]
-      ruby: ["ruby-2.7"]
-      mongodb-version: '4.7'
-      topology: '*'
```

Comment:
The load-balanced part of this asterisk is the one I am using.

Reply:
Added back.